install/install.py

```diff
@@ -113,6 +129,16 @@ def install(cmd_arguments):
     cp = ConfigParser.ConfigParser({"RootDir": root_dir, "Pid": pid})
     cp.read(distros_cfg)
 
+    # Check if the current distribution is compatible 
+    if not is_compatible():
```

Comment:
This is not needed. We do support non-Debian based distro (Archlinux) and will support other distros. So remove this check.